presto-main/src/test/java/com/facebook/presto/operator/scalar/sql/TestMapTopNKeysFunction.java

```diff
@@ -37,6 +38,7 @@ public void testBasic()
     @Test
     public void testNonPositiveN()
     {
+        assertFunction("MAP_TOP_N_KEYS(MAP(ARRAY[100, 200, 300], ARRAY[4, 5, 6]), 0)", new ArrayType(INTEGER), asList());
         assertFunction("MAP_TOP_N_KEYS(MAP(ARRAY[1, 2, 3], ARRAY[4, 5, 6]), -1)", new ArrayType(INTEGER), asList(1));
```

Comment:
Sure let me add that.

Reply:
So just to get it right, this is only for keys. So I have added both empty keys and null map in the test cases. We cannot have nulls in the key and since we're not looking for values so not sure if its gonna create any difference.